src/main/java/net/minecraftforge/common/config/Configuration.java

```diff
@@ -60,7 +63,13 @@
     public static final CharMatcher allowedProperties = CharMatcher.JAVA_LETTER_OR_DIGIT.or(CharMatcher.anyOf(ALLOWED_CHARS));
     private static Configuration PARENT = null;
 
-    File file;
+    @Deprecated
+    private File file;
+
+    @Deprecated
+    private CharSource staticSource;
+    @Deprecated
+    private CharSink staticSink;
```

Comment:
To be removed?
As i said a minute ago, main point of PR is to get rid of static (to one instance) source fields, but i'm keeping them for  those who want to load from same place without passing it each time to load & save.